cmd/shield/main.go

```diff
@@ -108,12 +112,15 @@ func main() {
 
 	c.Dispatch("help", "Show the list of available commands",
 		func(opts Options, args []string) error {
-			fmt.Fprintf(os.Stderr, "shield\n")
-			fmt.Fprintf(os.Stderr, "USAGE: shield [options] <command>\n\n")
-			fmt.Fprintf(os.Stderr, "Recommended:\n")
-			fmt.Fprintf(os.Stderr, "  Use SHIELD_TARGET environmental variable to store shield's IP:port.\n\n")
-			fmt.Fprintf(os.Stderr, "Commands:\n")
+			fmt.Fprintf(os.Stderr, "\n\x1b[1mNAME:\x1b[0m\n  shield\t\tCLI for interacting with the Shield API.\n")
+			fmt.Fprintf(os.Stderr, "\n\x1b[1mUSAGE:\x1b[0m\n  shield [options] <command>\n")
+			fmt.Fprintf(os.Stderr, "\n\x1b[1mENVIRONMENT VARIABLES:\x1b[0m\n")
+			fmt.Fprintf(os.Stderr, "  SHIELD_API\t\tused to specify the shield API's IP:port.\n")
+			fmt.Fprintf(os.Stderr, "  SHIELD_TRACE\t\tset to 'true' for trace output.\n")
+			fmt.Fprintf(os.Stderr, "  SHIELD_DEBUG\t\tset to 'true' for debug output.\n")
+			fmt.Fprintf(os.Stderr, "\n\x1b[1mINFO:\x1b[0m\n")
```

Comment:
Don't use raw ANSI escape codes unilaterally.  (see later comment for more details)